### DISCUSSION TITLE
[EXPLOIT] GoGuardian TABCRASH (@py660)

### DISCUSSION THREAD

**Discussion by py660** [3kh0/ext-remover#861] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
# Teacher's can still see your screen, but they can't block or close any of your tabs.

1. create a bookmark: `javascript: window.onbeforeunload = ()=>{return false;}`
2. Hold down CTRL and then SPAM CLICK the bookmark until you're well above the tab limit, opening a bunch of `about:blank` pages. 3. It might ask if you want to leave this page, this is goguardian trying to close it. Say no.
Enjoy your unblocked stay!

**Comment by py660** [comment]
@3kh0 Add this plz?

**Reply by 3kh0** [reply]
Send a pull request and I'll merge it

**Reply by py660** [reply]
cool

**Reply by py660** [reply]
just did :+1:
